Lesson_13/task3_xml.py

```diff
@@ -0,0 +1,62 @@
+"""Create search function for XML files.
+
+For the ideas_for_test/work_with_xml/groups.xml file,
+create a group/number search function and return the timingExbytes/incoming
+value, output the result to the console through the logger at the info level.
+"""
+from pathlib import Path
+
+from defusedxml.ElementTree import ParseError, parse
```

Comment:
Ok but then you need to add requirements.txt to your pull request of leave instruction what should be installed additionally. Try to avoid silent adding of new modules

Reply:
updated with commit e7ce4b62b49a2daa3850be70c2083336cf6a52d8